.travis.yml

```diff
@@ -2,14 +2,8 @@ language: go
 services:
   - redis-server
 go:
-  - 1.4
   - 1.5
+  - 1.6
   - tip
-before_install:
```

Comment:
Yea I'll have to screw with this again. I removed it temporarily so we could see Travis status.